assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -30,10 +30,6 @@ class TextBlockEdit extends Component {
 		this.onReplace = this.onReplace.bind( this );
 	}
 
-	componentDidMount() {
-		maybeUpdateFontSize( this.props );
```

Comment:
We should probably try find the cause for the font size difference instead of removing the initial calculation. As mentioned, this seems to cause issues with templates.

Reply:
Reverted it for now.